components/edit-collective/sections/Tiers.js

```diff
@@ -25,38 +25,35 @@ import { listTierQuery } from '../tiers/EditTierModal';
 import { collectiveSettingsV1Query } from './EditCollectivePage';
 
 // TODO Make this a common function with the contribute section
-const getFinancialContributions = (collective, sortedTiers) => {
-  const hasCustomContribution = !get(collective, 'settings.disableCustomContributions', false);
-  const waysToContribute = [];
+const getTiers = (collective, sortedTiers, intl) => {
+  const defaultContributionEnabled = !get(collective, 'settings.disableCustomContributions', false);
 
-  sortedTiers.forEach(tier => {
-    if (tier === 'custom') {
-      if (hasCustomContribution) {
-        waysToContribute.push({
+  const createdTierCards = sortedTiers.map(tier => ({
+    key: tier.id,
+    Component: ContributeTier,
+    componentProps: {
+      collective,
+      tier,
+      hideContributors: true,
+      hideCTA: true,
+    },
+  }));
+  const additionalTierCards = defaultContributionEnabled
+    ? [
+        {
           key: 'custom',
           Component: ContributeCustom,
           componentProps: {
             collective,
             hideContributors: true,
             hideCTA: true,
+            missingCTAMsg: intl.formatMessage({ defaultMessage: 'The default contribution tier cannot be edited.' }),
           },
-        });
-      }
-    } else {
-      waysToContribute.push({
-        key: tier.id,
-        Component: ContributeTier,
-        componentProps: {
-          collective,
-          tier,
-          hideContributors: true,
-          hideCTA: true,
         },
-      });
-    }
-  });
+      ]
+    : [];
 
-  return waysToContribute;
+  return [...createdTierCards, ...additionalTierCards];
```

Comment:
I would put it at the beginning so that you can instantly see the effects of your changes - even if there's a long list of tiers ahead.

```suggestion
  return [...additionalTierCards, ...createdTierCards];
```

Reply:
I sorted by the user-maintained sorting instead.

Reply:
@Abekonge That's even better